.github/workflows/go.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.19
+        go-version: 1.21-bookworm
```

Comment:
This looks like a container image name, but the setup-go action doesn't work by pulling a container image.  My understanding is that all the legal versions are listed in https://github.com/actions/go-versions/blob/main/versions-manifest.json.